sites/public/pages/listings.tsx

```diff
@@ -100,7 +108,8 @@ const ListingsPage = () => {
   )
 
   const numberOfFilters = filterState
-    ? Object.keys(filterState).filter((p) => p !== "$comparison").length
+    ? // "availability" is a convenience field only used by the frontend. Don't track it here.
+      Object.keys(filterState).filter((p) => p !== "$comparison" && p !== "availability").length
```

Comment:
I don't understand why filtering it here doesn't mean it won't show up in the number of filters? Is it because the availability field of the filterState is always set? Can you make the empty value represent the 'any' filter type instead, and then you wouldn't need to filter it out here?

Reply:
This was originally because the `availability` filter wasn't going to be sent to the backend, only the child filters were. But now I'm sending availability down so this is resolved.